templates/repo/pulls/fork.tmpl

```diff
@@ -53,20 +53,18 @@
 					</div>
 					<div class="inline field">
 						<label>{{ctx.Locale.Tr "repo.fork_branch"}}</label>
-						<div class="ui selection dropdown">
+						<div class="ui selection dropdown ellipsis-items-nowrap">
 							<input type="hidden" id="fork_single_branch" name="fork_single_branch" value="" required>
-							<span class="text truncated-item-container" data-value="" title="{{ctx.Locale.Tr "repo.all_branches"}}">
```

Comment:
`truncated-item-container` is deprecated?
If it is, then I can image all places using it have the same problem.
e.g.
![image](https://github.com/user-attachments/assets/3f21c27b-f114-43e7-b538-81200a92e1e1)